### PR TITLE
♿ TetraLogical: AMP Stories accessibility review / documentation

### PR DESCRIPTION
**Pull request by TetraLogicalHelpdesk**
Suggested changes and additions based on the TetraLogical accessibility review commissioned by @nainar / @caroqliu

Main changes:

- added a few minor clarifications, where appropriate, advising authors to consider use information conveyed through audio, judicious use/avoiding overuse of animations, suggested contrast ratios for text over backgrounds.
- reworked the explanation of fallback content - not just as a way to show a "your browser doesn't support this", but an opportunity to still convey useful information to users
- emphasised that allowing landscape mode (rather than just showing a door-slam "tilt your device" type message) is preferable (as this *would* otherwise contravene https://www.w3.org/TR/WCAG/#orientation ... but probably something that AMP Stories should address at a more fundamental/systemic level, as noted in the recent accessibility review)
- fixes to visually broken pages (non-working "Develop > Animate & Transition" in the left-hand navigation, missing left-hand sidebar in https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/discovery_web_stories/?format=stories, broken bottom part of the page in https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design/?format=stories)
- rewritten text alternatives for meaningful images/illustrations in the documentation where necessary
- added placeholder `alt="..."` to most code sample examples of images, as this increases the chance that authors will actually remember to use the attribute to provide text alternatives
- small fixes to typos found along the way

x-ref: https://github.com/ampproject/amphtml/pull/32779

### CODE REVIEW

**Comment by patrickkettner** [comment]
Thanks so much, @patrickhlauke! @nainar @CrystalOnScript As these are stories, does anyone specific need to sign off?￼

File: examples/source/1.components/amp-story-interactive-poll/index.html [hunk 1]
```diff
@@ -192,7 +196,11 @@
 </html>
 
 <!--
-        # More resources
+  ## Accessibility considerations
 
-        If you want to learn more about all the interactive experiences for Web Stories, head over to the [documentation](/documentation/components/amp-story-interactive?format=stories) or check out the guide on how to make a [story with quizzes](/documentation/examples/components/amp-story-interactive-quiz/?format=stories) instead.
-      -->
+  Currently, this component lacks keyboard and screen reader accessibility.
```
Comment (newmuis) [inline examples/source/1.components/amp-story-interactive-poll/index.html:201]: Can we add a link to https://github.com/ampproject/amphtml/issues/32751?  That should hopefully get resolved soon, but good to have visibility.

File: examples/source/1.components/amp-story-interactive-quiz.html [hunk 2]
```diff
@@ -218,6 +222,10 @@
 </html>
 
 <!--
+  ## Accessibility considerations
+
+  Currently, this component lacks keyboard and screen reader accessibility.
```
Comment (newmuis) [inline examples/source/1.components/amp-story-interactive-quiz.html:227]: Same as above (link to https://github.com/ampproject/amphtml/issues/32751)

File: examples/source/visual-effects/AMP_Story_Animations.html [hunk 3]
```diff
@@ -8,6 +8,8 @@
   ## Introduction
 
   AMP Stories support animations out-of-the-box.  Every element inside an `amp-story-page` can define an entrance animation. You can find a list of all supported animations [here](/content/amp-dev/documentation/components/reference/amp-story.md#animations).
+
+  [tip type="note"]Animations can help make your Web Story more visually exciting and engaging, but use them sparingly. Some users may find long, continuous animations distracting. Other users may have motion sensitivity and be adversely affected by excessive use of motion and parallax effects.[/tip]
```
Comment (newmuis) [inline examples/source/visual-effects/AMP_Story_Animations.html:12]: Can we add a link to https://github.com/ampproject/amphtml/issues/24821, which tracks respecting the `prefers-reduced-motion` directive for such users?

File: examples/source/visual-effects/Ken_Burns.html [hunk 4]
```diff
@@ -63,12 +63,14 @@
     - Animate the `<amg-img>` element using the `animate-in="pan-right"`. (You can also use `pan-left`, `pan-up`, and `pan-down`). [See amp-story animations list.](/documentation/examples/visual-effects/amp_story_animations/?format=stories#zoom-in)
 
     Tip: It's important that the **image** sets the **panning animation**, whereas the image **container** sets the **zoom animation**. Otherwise the dimensions of the image will change, causing undesired results in the animation.
+    
+    Tip: Animations can help make your Web Story more visually exciting and engaging, but use them sparingly. Some users may find long, continuous animations distracting. Other users may have motion sensitivity and be adversely affected by excessive use of motion and parallax effects.
```
Comment (newmuis) [inline examples/source/visual-effects/Ken_Burns.html:67]: Can we add a link to ampproject/amphtml#24821, which tracks respecting the prefers-reduced-motion directive for such users?

File: pages/content/amp-dev/documentation/guides-and-tutorials/start/visual_story/animating_elements.md [hunk 5]
```diff
@@ -125,6 +125,10 @@ To apply an animation entrance to an element, you must specify the <code>animate
 Explore the different animation effects by adding the `animate-in="<animation preset>"` attribute to elements on your story pages.
 [/tip]
 
+[tip type="note"]
+Animations can help make your Web Story more visually exciting and engaging, but use them sparingly. Some users may find long, continuous animations distracting. Other users may have motion sensitivity and be adversely affected by excessive use of motion and parallax effects.
```
Comment (newmuis) [inline pages/content/amp-dev/documentation/guides-and-tutorials/start/visual_story/animating_elements.md:129]: Can we add a link to ampproject/amphtml#24821, which tracks respecting the prefers-reduced-motion directive for such users?

**Review by newmuis — COMMENTED** [review]
Looks mostly good to me for stories content, though I've added a few suggestions for how we can link tracking issues for bugfixes that are in progress.

There's quite a bit here that's outside the context of stories, that I'm probably not the right person to review.

File: examples/source/visual-effects/Ken_Burns.html (hunk 4 above)
Comment (patrickhlauke) [reply examples/source/visual-effects/Ken_Burns.html:67]: as far as i can tell, these animation effects already respect that setting. but not all users will have that setting always turned on, and this is still good advice in general about not overdoing it in general, I'd say.

File: examples/source/1.components/amp-story-interactive-poll/index.html (hunk 1 above)
Comment (patrickhlauke) [reply examples/source/1.components/amp-story-interactive-poll/index.html:201]: Do you want @TetraLogicalHelpdesk to make the suggested changes, or will you/the team do these additions before merging?

File: examples/source/1.components/amp-story-interactive-quiz.html (hunk 2 above)
Comment (mszylkowski) [reply examples/source/1.components/amp-story-interactive-quiz.html:227]: Currently working on the PR to make it accessible, so no need to add it to the documentation

File: examples/source/1.components/amp-story-interactive-poll/index.html (hunk 1 above)
Comment (mszylkowski) [reply examples/source/1.components/amp-story-interactive-poll/index.html:201]: Currently working on the PR to make it accessible, so no need to add it to the documentation

**Comment by TetraLogicalHelpdesk** [comment]
Note that it's probably best at this stage (since the project relating to this was closed quite some time ago) if somebody on your side could shepherd this PR through to completion. Alternatively, we can add some extra time to the next AMP-related project to take care of this? Let us know how you'd like to proceed. /cc @nainar @caroqliu

**Comment by caroqliu** [comment]
> Note that it's probably best at this stage (since the project relating to this was closed quite some time ago) if somebody on your side could shepherd this PR through to completion. Alternatively, we can add some extra time to the next AMP-related project to take care of this? Let us know how you'd like to proceed. /cc @nainar @caroqliu

Happy to take over, thanks for getting this rolling!

**Comment by caroqliu** [comment]
Closing in favor of #5970 since I didn't have permissions to push directly to this branch.

**Pull request closed without merging**